include/configcat/configcatoptions.h

```diff
@@ -21,6 +21,11 @@ struct ProxyAuthentication {
     std::string password;
 };
 
+struct SslOptions {
+    std::optional<bool> verifySSL;
```

Comment:
This parameter would allow the SDK consumer to enable/disable the SSL verification of the peer & host.

I could simplify it to a `bool verifySSL = true` if the `std::optional<>` is overengineering.

Or we could remove it altogether if we don't consider it a valuable option (or too much of a security risk).

Reply:
Option was removed